src/components/Composer/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
@thanhsonng Why should we disable this rule?

Reply:
@namdaoduy This rule causes linting error in these files and thus prevents new commit to be made.
I just pushed a new commit that disables this rule globally, as per your recommendation.